conjur/cli.py

```diff
@@ -69,6 +78,10 @@ def run(self, *args):
         Main entrypoint for the class invocation from both CLI, Package, and
         test sources. Parses CLI args and invokes the appropriate client command.
         """
+        formatter_class = lambda prog: argparse.RawTextHelpFormatter(prog,
```

Comment:
please remove and rebase with master. My PR was merged that has these changes so you don't need to add this by yourself